packages/input-container/theme/lumo/vaadin-input-container-styles.js

```diff
@@ -8,14 +8,14 @@ registerStyles(
   'vaadin-input-container',
   css`
     :host {
-      border-radius: var(--lumo-border-radius-m);
       background-color: var(--lumo-contrast-10pct);
       padding: 0 calc(0.375em + var(--lumo-border-radius-m) / 4 - 1px);
       font-weight: 500;
       line-height: 1;
       position: relative;
       cursor: text;
       box-sizing: border-box;
+      --_border-radius: var(--lumo-border-radius-m);
```

Comment:
I don't approve this approach, as it makes the `--_border-radius` property seem public, which it isn't. We should find a way to do this instead:

```css
html {
  --vaadin-input-container-border-radius: var(--lumo-border-radius-m);
}
```

Reply:
Good point. Should it be placed in some new style module under Lumo styles?

Reply:
Moved the style above to `style.js` placed in `vaadin-lumo-styles` as discussed internally.